examples/tracing.rb

```diff
@@ -11,9 +11,10 @@
   #exec '/bin/bash', '-l'
 end
 
+p $?.inspect
 ret = Seccomp.start_trace(pid) do |syscall, _pid, ud|
   name = Seccomp.syscall_to_name(syscall)
   puts "[#{_pid}]: syscall #{name}(##{syscall}) called. (ud: #{ud})"
 end
 
-p(ret)
+p $?.exited?
```

Comment:
Ah, this example looks broken...

(I was ok that $? would be set by Process::Status too)

Reply:
oh... I fixed at ce436c4